internal/pkg/resource/resource.go

```diff
@@ -15,7 +15,7 @@ const (
 	ConsumerShare         = "consumer share"
 	Context               = "context"
 	Environment           = "environment"
-	FlinkComputePool      = "Flink compute pool"
+	FlinkComputePool      = "flink compute-pool"
```

Comment:
> I think the dash is nice

I don't see a dash in any of your team's documentation. Are you sure it's a "Flink compute-pool" and not a "Flink compute pool"? (I don't think this needs to change)

Reply:
We have to write it exactly like this or else the auto-generated suggestions won't be correct. See this example when it's only "compute pool":

<img width="1087" alt="image" src="https://github.com/confluentinc/cli/assets/11739405/40f7cc83-4939-40e9-a428-438dd080e226">

conlfuent Flink compute-pool list would also not work. The other resources in uppercase here can only be uppercase because they aren't used for error handling/don't generate suggestions.

Reply:
Oh I see... hmm that's a bit tricky then, because in the messages the user sees it looks weird. E.g. in the tests for deleting a compute pool: 
`Deleted flink compute-pool "lfcp-123456".`

I suppose we would have to find a way to differentiate between the display names and the commands in CatchCCloudV2ResourceNotFoundError function, or maybe Brian knows a better way to do this 🤔

Reply:
Actually just found an example, check out the `CatchKafkaNotFoundError` [here](https://github.com/confluentinc/cli/blob/deac7c07364cc73ebb7f9c0ba477807450d317d1/internal/pkg/errors/catcher.go#L210). There they are also differentiating between the suggestions and the display name of the resource, so you could add a custom catcher for compute pools `CatchFlinkComputePoolNotFoundError`

Reply:
Done: 

<img width="1059" alt="image" src="https://github.com/confluentinc/cli/assets/11739405/fad02d75-eff0-4a77-b32f-4314eececf96">

Reply:
> We have to write it exactly like this or else the auto-generated suggestions won't be correct.

I think you may have already come to this conclusion, but these constants are used for human-readable text, not for formatting commands.